AWS/CDK/cdk.json

```diff
@@ -0,0 +1,17 @@
+{
+  "app": "node bin/cdk.js",
+  "context": {
+    "@aws-cdk/aws-apigateway:usagePlanKeyOrderInsensitiveId": true,
+    "@aws-cdk/core:enableStackNameDuplicates": "true",
+    "aws-cdk:enableDiffNoFail": "true",
+    "@aws-cdk/core:stackRelativeExports": "true",
```

Comment:
Why are three `"true"` and the rest are `true`?

Reply:
these are the defaults for "cdk init" - I didn't bother to tweak them.

Reply:
Super odd